ansible_base/lib/utils/views/django_app_api.py

```diff
@@ -28,5 +29,15 @@
         logger.exception(_("Failed to import {parent_view}, defaulting to AnsibleBaseView".format(parent_view=parent_view)))
 
 
+# In case an app is using features like authentication, resources, or RBAC
+# but not using rest_filters app, we need to specify the same filter backends
+dab_filter_backends = list(api_settings.DEFAULT_FILTER_BACKENDS)
+
+for backend_name in settings.ANSIBLE_BASE_CUSTOM_VIEW_FILTERS:
+    backend = import_from_string(backend_name, 'ANSIBLE_BASE_CUSTOM_VIEW_FILTERS')
+    if backend not in dab_filter_backends:
+        dab_filter_backends.append(backend)
```

Comment:
Since these are in settings and unlikely to change w/o a reboot should we cache this somehow (maybe lru) so we aren't computing this over and over for every view?

Reply:
Also, the way this is implemented, if we do have `ansible_base.rest_filters` enabled this logic would still kick in right? That seems undesirable.

Reply:
> Since these are in settings and unlikely to change w/o a reboot should we cache this somehow (maybe lru) so we aren't computing this over and over for every view?

This should only get ran on import, and only once, with the way python imports work. Yes, _most_ views should inherit from this class, but the type definition of this class should only happen once.

Reply:
I previously pushed some logic to address the concern I think you expressed, so I just now pushed a test for that as well. We have a condition `if backend not in dab_filter_backends:`, so if the "rest_filters" includes an entry via settings (or the user does this) then it will not duplicate that entry in the list. What we get is a merged list of the backends from settings and the declared backends we want for our own DAB views via `ANSIBLE_BASE_CUSTOM_VIEW_FILTERS`. Let me know if there are any other questions, I think the concerns here have all been addressed.